gcp/modules/couchdb/main.tf

```diff
@@ -3,13 +3,47 @@ terraform {
 }
 
 variable "values_dir" {}
+variable "release_namespace" {
+	default = "gpii"
+}
 
 module "couchdb" {
   source = "/exekube-modules/helm-template-release"
 
   release_name      = "couchdb"
-  release_namespace = "gpii"
+  release_namespace = "${var.release_namespace}"
   release_values    = "${var.values_dir}/couchdb.yaml"
 
   chart_name = "../../../../../charts/couchdb"
 }
+
+variable "couchdb_admin_username" {}
+variable "couchdb_admin_password" {}
+
+resource "null_resource" "couchdb_finish_cluster" {
+  depends_on = ["module.couchdb"]
+
+  provisioner "local-exec" {
+    command = <<EOF
+      while [ "$PODS_READY" != "true" ]; do
```

Comment:
This could loop infinitely. Will terraform timeouts (or timeouts from some other level) prevent a rake run from being stuck forever?

Reply:
Added timeout to Terraform resource.

Reply:
Hmm, I don't see it in the diffs? Am I missing something?